fennel/client_tests/test_social_network.py

```diff
@@ -151,8 +150,6 @@ def extractor_category_view(
         )
 
 
-@pytest.mark.skip(reason="Failing in main currently: TODO: DEV-2550")
-@pytest.mark.slow
```

Comment:
Lets keep the @pytest.mark.slow marker, to make iteration speed faster while running tests locally

Reply:
alright